vm/os-windows.cpp

```diff
@@ -185,12 +185,14 @@ uint64_t nano_count() {
 
 void sleep_nanos(uint64_t nsec) { Sleep((DWORD)(nsec / 1000000)); }
 
+#ifndef EXCEPTION_DISPOSITION
```

Comment:
Why is this ``ifndef`` added?

Reply:
It probably prevents an "illegal re-declaration of enum EXCEPTION_DISPOSITION" when MinGW pre-defines it

Reply:
That's exactly right. MinGW predefines it using defines. This is the simplest way to adapt for that.